howso/utilities/feature_attributes/abstract_data.py

```diff
@@ -298,6 +298,10 @@ def _get_random_value(self, feature_name: str, no_nulls: bool = False) -> t.Any
         """
         return self.data.get_random_value(feature_name, no_nulls=no_nulls)
 
+    def _get_unique_count(self, feature_names: str | Iterable[str]) -> int:
```

Comment:
This uses a different parameter name than the base class, lets make them consistent.

Reply:
Same in the pandas version